app/MMB/src/navigation/DetailScreen.js

```diff
@@ -104,6 +105,10 @@ export default class DetailsScreen extends React.Component<Props, State> {
     };
   };
 
+  componentDidMount = () => {
+    this.props.setIsPastBooking(this.props.isPastBooking);
```

Comment:
We could probably add there even the `bookingId` so we don't have to prop-drill it, right? (not in this PR)

Reply:
exactly, that is an option 🙂